letta-ai/letta
# WIP - dmg/exe installable Letta

**Please describe the purpose of this pull request.**
This PR will add installable artifacts (dmg and exe) to the CI cycle. they can then be presented as "download Letta" links

**How to test**
In Phases: 
1. ✅ _embedded pg_: install from `./installable_apps/requirements.txt` and then run `./installable_apps/startup.py`. Visit [http://localhost:8283](http://localhost:8283) and create a new agent, verify that agent saves. Start and stop the service, confirm that the agent persisted.
2. ✅  _build installable for MacOS_: See [Notion](https://www.notion.so/DMG-EXE-distribution-model-design-doc-1430ee6f3350805e892bfe4c8f9e9d14?pvs=4) for instructions.
3. ❌ _build installable for Windows_: Future PR
4. ❌ _CI release_: Future PR
